pkg/machine/wsl/wutil/wutil.go

```diff
@@ -114,13 +91,50 @@ func IsWSLStoreVersionInstalled() bool {
 	return true
 }
 
-func matchOutputLine(output io.ReadCloser, match string) bool {
-	scanner := bufio.NewScanner(transform.NewReader(output, unicode.UTF16(unicode.LittleEndian, unicode.UseBOM).NewDecoder()))
-	for scanner.Scan() {
-		line := scanner.Text()
-		if strings.Contains(line, match) {
-			return true
-		}
+func existsKernel() bool {
```

Comment:
would you mind a little more concise naming of this function? like WSL something or another.

Reply:
Do you have any recommendations? Or move the code to the `IsWSLInstalled` function?